cli/app.go

```diff
@@ -942,6 +942,25 @@ After creation, use 'viam module update' to push your new module to app.viam.com
 					},
 					Action: UpdateModuleAction,
 				},
+				{
+					Name:  "update-models",
```

Comment:
for documentation: this command will only work if you have a binary that will run on this os...e.g. command won't work on windows even though cli will. worth dropping a comment here with this info.

Reply:
added note